src/component/axis/AxisBuilder.ts

```diff
@@ -776,9 +781,18 @@ function buildAxisLabel(
 
         const tickCoord = axis.dataToCoord(tickValue);
 
+        const font = labelModel.getFont();
+        const labelFormatter = makeLabelFormatter(axis);
+        const rect = textContain.getBoundingRect(
```

Comment:
`textContain.getBoundingRect` won't consider rich text. Instead, we can always use `textEl.getBoundingRect()` to get the accurate rect. Then we can calculate the `y`.